Update Lighthouse Book for Electra features (#7280)
sigp/lighthouse: book/src/validator_consolidation.md

# Consolidation

> With [Pectra](https://ethereum.org/en/history/#pectra) upgrade, a validator can hold a stake of up to 2048 ETH. This is done by updating the validator withdrawal credential to type 0x02. With 0x02 withdrawal credential, it is possible to consolidate two or more validators into a single validator with a higher stake.

Reviewer: ```suggestion
With the [Pectra](https://ethereum.org/en/history/#pectra) upgrade, a validator can hold a stake of up to 2048 ETH. This is done by updating the validator withdrawal credentials to type 0x02. With 0x02 withdrawal credentials, it is possible to consolidate two or more validators into a single validator with a higher stake.
```

Let's take a look at an example: Initially, validators A and B are both with 0x01 withdrawal credentials with 32 ETH. Let's say we want to consolidate the balance of validator B to validator A, so that the balance of validator A becomes 64 ETH. These are the steps:

> 1. Update the withdrawal credential of validator A to 0x02. You can do this using [Siren](./ui.md) or the [staking launchpad](https://launchpad.ethereum.org/en/). Select:

Reviewer: ```suggestion
1. Update the withdrawal credentials of validator A to 0x02. You can do this using [Siren](./ui.md) or the [staking launchpad](https://launchpad.ethereum.org/en/). Select:
```

    - source validator: validator A
    - target validator: validator A

>     > Note: After the update, the withdrawal credential type 0x02 cannot be reverted to 0x01, except that the validator exits and make a fresh deposit.

Reviewer: ```suggestion
    > Note: After the update, the withdrawal credential type 0x02 cannot be reverted to 0x01, unless the validator exits and makes a fresh deposit.
```
